### ISSUE TITLE
Duplicate Function Names & Build Tags

### ISSUE THREAD

**Issue by jon-mcmillan** [nelsam/vidar#19]
It's possible to have a valid package with multiple versions of the same func by using build tags.

**Expected Behavior**
List all versions of package variables, functions, etc.
![screen shot 2015-12-27 at 7 33 43 pm](https://cloud.githubusercontent.com/assets/397948/12012767/d7ca12b8-acd1-11e5-8ce2-b55a78177130.png)

**Actual Behavior**
App crashes.

```
panic: Adapter for control '*basic.Window > *commander.Commander > *controller.Controller > *navigator.Navigator > *mixins.LinearLayout > *navigator.dirTree' returned duplicate item
```

**Comment by nelsam** [comment]
Solved with 5cb7d4b2060279a4b4fb475e32b0d9226d3057bf

- Both `_` variables and `init` functions are now ignored by the table of contents.
- Logic in files with build tags (explicit or implicit) have their tags displayed sort of like the following:
```
consts:
  - foo (linux)
  - foo (windows)
```

The syntax for the build tag stuff in the parens matches [the go build documentation for build constraints](https://golang.org/pkg/go/build/#hdr-Build_Constraints).  Multiple lines will be consolidated, so:
```
// +build linux darwin
// +build amd64 x86
```
becomes `(linux,amd64 linux,x86 darwin,amd64 darwin,x86)`.  It's a bit wordy, but I can't think of anything more sane yet.
